src/client/gui/lib/extensions.dart

```diff
@@ -107,3 +107,45 @@ extension NullableMap<T> on T? {
     }
   }
 }
+
+/// A custom wrapper for SelectableText with a white popup (right-click) menu.
+class WhitePopupMenuSelectableText extends StatelessWidget {
```

Comment:
That's kind of a lengthy name IMO. You can name it just `SelectableText` and use `import 'package:flutter/material.dart' hide SelectableText;` so that you can use yours without interfering with Flutter's.
Also, you can move this to its own file. `extensions.dart` is more for defining useful extension methods on types rather that for widgets.

Reply:
> You can name it just `SelectableText` and use `import 'package:flutter/material.dart' hide SelectableText;` so that you can use yours without interfering with Flutter's.

When I tried to do this I wasn't able to define the contextMenuBuilder parameter: `The contextMenuBuilder parameter is not defined in the SelectableText widget.` 
So instead I moved the class into its own file at selectable_text.dart and renamed it to WhiteSelectableText

Reply:
Yeah, it was because your `SelectableText` was conflicting with Flutter's here. You can circumvent this by importing Flutter under an alias in just this file. Take a look [here](https://github.com/canonical/multipass/blob/main/src/client/gui/lib/tooltip.dart) where there is a custom `Tooltip` widget that wraps Flutter's `Tooltip`.

Reply:
I understand if this feels weird, and maybe it creates confusion. But IMO it's better than coming up with more elaborate names for widgets just because Flutter used them first. I'm open to criticism though!